core/backend/src/ElementTreeWalker.ts

```diff
@@ -50,6 +50,8 @@ enum ElementPruningClassification { PRUNING_CLASS_Normal = 0, PRUNING_CLASS_Subj
 
 function classifyElementForPruning(iModel: IModelDb, elementId: Id64String): ElementPruningClassification {
   const el = iModel.elements.getElement(elementId);
+  if (el instanceof DefinitionContainer)
+    return ElementPruningClassification.PRUNING_CLASS_DefinitionPartition;
   return (el instanceof Subject) ? ElementPruningClassification.PRUNING_CLASS_Subject :
     (el instanceof DefinitionElement) ? ElementPruningClassification.PRUNING_CLASS_Definition :
       (el instanceof DefinitionPartition) ? ElementPruningClassification.PRUNING_CLASS_DefinitionPartition :
```

Comment:
Could you group these as below?

```suggestion
      (el instanceof DefinitionPartition || el instanceof DefinitionContainer) ? ElementPruningClassification.PRUNING_CLASS_DefinitionPartition :
```

Reply:
Feel free to resolve both I left if not. Just wanted to make it possible for the suggestions to be committed from github and have it be good to go

Reply:
No. DefinitionContainer is-a DefinitionElement, so the (el instanceof DefinitionElement) ? ElementPruningClassification.PRUNING_CLASS_Definition :  case would grab it.

Reply:
Ah got it. Could you add a comment explaining that?

Reply:
done

Reply:
thank you